ansible/playbooks/roles/preflight_facts/tasks/kubernetes/get-registered-masters.yml

```
@@ -17,15 +17,19 @@
 
     - name: Get list of all master nodes
       when: stat_kubeconfig.stat.exists
-      command: |
-        kubectl get nodes \
-          --selector={{ selector }} \
-          --output=jsonpath={{ jsonpath }}
+      # When Epiphany doesn't support K8s versions with old label,
+      # can be changed to command: "kubectl get nodes --selector={{ selector }} --output=jsonpath={{ jsonpath }}"
+      shell: |-
+        kubectl get nodes --output=json \
+          | jq --raw-output {{ query }}
+      args:
+        executable: /bin/bash
       vars:
-        selector: >-
-          'node-role.kubernetes.io/master'
-        jsonpath: >-
-          '{range .items[*]}{.metadata.name}{"\n"}{end}'
+        query: >-
```

```suggestion
        _query: >-
```
Let's use _ prefix for local task's vars since they can be overridden by for example `set_fact`.